lsp-clients/n4js-vscode-extension/src/ExtensionProvider.n4js

```diff
@@ -93,19 +93,37 @@ export function getActivate(vscode: any+, vscodeLC: any+): any {
         // Create the language client and start the client.
         let lc = new vscodeLC.LanguageClient(CHANNEL_NAME, serverOptions, clientOptions, true);
         lc.onReady().then(() => {
-            const requestType = new vscodeLC.RequestType ('n4/documentContents');
+
+            
+            const rtDocumentContents = new vscodeLC.RequestType('n4js/documentContents');
```

Comment:
documentContents will also return content for `n4ts` files from the buildin-scheme. I don't think it matters much, though.

Reply:
I wanted to avoid 'n4' as a term, that is why I renamed it. An argument for the term 'n4js' at this location could be that 'n4js' not only denotes the file extension but also the project name and the language as a whole.